### PR TITLE
SCAN - TeX output to XGP scan file

### PR DESCRIPTION
**Pull request by larsbrinkhoff**
SYSEN1; SCAN 561  
SYS3; TS SCAN

### CODE REVIEW

**Comment by atsampson** [comment]
The input format this is expecting is XGP text with control codes - so it's basically doing the same job as VERSA. They support different subsets of XGP features, though!

This implies that ITS TeX produced XGP text output directly...

File: src/sysen1/scan.561 [hunk 1]
```diff
@@ -0,0 +1,3016 @@
...
+pktdes: 0			   ; ITS fills in this
+pktsrc:	0			   ; ITS fills in this
+pktnum:	0			   ; ITS fills in this
+pktdat:	block 172		   ; data area of packet buffer
+pktend::			   ; end of packet
+
+ctlpkt:	block 126.		   ; control packet.
+
+ife $debug,[
+outdev:	sixbit /AI/
```
[the remark this answers is not in the document]
Comment (larsbrinkhoff) [reply src/sysen1/scan.561]: I'm tempted so change outdev and quedev to DSK, but maybe it's better to wait.  Maybe by the time we want to run this, AI has come up again.

**Comment by larsbrinkhoff** [comment]
Written by @BobKerns.

See messages in thread starting here: https://twitter.com/BobKerns/status/1458536593648586753

**Comment by BobKerns** [comment]
I would never have recognized the file by the name. It makes for interesting reading, especially WRT what I remember and what I don't.

I've forgotten a lot of the less-used PDP-10 instructions, system calls, everything about communicating with the XGP. I don't remember what prompted me to write this. But I remember trying to write object-oriented remote software in assembly!

Thanks for putting 2 and 2 together here.

**Pull request merged**